### ISSUE TITLE
Adding new disclosures for v. 5 option Plans

### ISSUE THREAD

**Issue by g4gaurang** [willdwyer/bcbsmaissuestracker#252]
See email attachment with new disclosures for v. 5 option Plans. These need to be appropriately stitched to the applicable standard 2016 SBCs in the system.  The mapping is as follows - 

55-1251 HMO Blue Local Option Plans
55-1252 HMO New England Option Plans
55-1253 Preferred Blue Option Plans

These are in lieu of the v.4 option Plans disclosures. So you will need to swap those out for these on the applicable standard 2016 SBCs. Once you make the change, any 2016 Standard Options SBC document or its derivative custom SBC generated with disclosures through the system will include the appropriate v.5 disclosure.

So in summary, these – 
55-0405 HMO Blue Options v.4
55-0406  HMO Blue New England Options v.4.
55-0407 Preferred Blue PPO Options v.4

Need to be swapped out for these – 
55-1251 HMO Blue Local Option Plans
55-1252 HMO New England Option Plans
55-1253 Preferred Blue Option Plans

Let us know if you have any questions or doubts

**Comment by michparrish** [comment]
These documents have been set up in pclient5 and tested successfully. Please test there and once approved, we'll move to Prod.

**Comment by willdwyer** [comment]
Hello,

I tested in PCLIENT5, and the adjustment looks like it has been made.  Please move to production.

**Comment by michparrish** [comment]
Moved to Prod - please confirm and close item. thank you!

**Comment by willdwyer** [comment]
Hi Michelle,

This looks good, and is now being closed.
